src/Storages/MySQL/MySQLSettings.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <Core/Defines.h>
+#include <Core/BaseSettings.h>
+
+
+namespace Poco::Util
+{
+    class AbstractConfiguration;
+}
+
+
+namespace DB
+{
+class ASTStorage;
+
+#define LIST_OF_MYSQL_SETTINGS(M) \
+    M(UInt64, connection_pool_default_size, 1, "Minimal connection pool size", 0) \
```

Comment:
Actually for now it is better to not expose this setting to client, because we want to make generic pool for MySQL, PostgreSQL, ODBC and need to check which properties we want to expose. I am not sure that client want to configure connection_pool_default_size.

Reply:
OK, sounds reasonable (I've just export all available settings).

Reply:
@azat ok so you will remove connection_pool_default_size as setting ?

Reply:
Yep, done.